library/core/resource-loader/src/main/java/org/quiltmc/qsl/resource/loader/api/ResourcePackActivationType.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2021 QuiltMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.quiltmc.qsl.resource.loader.api;
+
+/**
+ * Represents the resource pack activation type.
+ */
+public enum ResourcePackActivationType {
+	/**
+	 * Normal activation. The user has full control over the activation of the resource pack.
+	 */
+	NORMAL,
+	/**
+	 * Enabled by default. The user has still full control over the activation of the resource pack.
+	 *
+	 * <p>Note: this setting can only be satisfied on data packs, client resource packs cannot be by default enabled.
```

Comment:
This is really nit-picky, but this word order is really awkward.
```suggestion
	 * Enabled by default. The user still has full control over the activation of the resource pack.
	 *
	 * <p>Note: this setting can only be satisfied on data packs, client resource packs cannot be enabled by default.
```